_test/setup.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+trap "exit 1" TERM
+export TOP_PID=$$
+NAMESPACE="${2:-pipelinelib-testing}"
+TRAVIS_REPO_SLUG="${3:-redhat-cop/pipeline-library}"
+TRAVIS_BRANCH="${4:-master}"
+
+cluster_up() {
+  set +e
+  built=false
+  while true; do
+    if [ -z $IP_ADDR ]; then
+      DEV=$(ip link | awk '/state UP/{ gsub(":", ""); print $2}')
+      IP_ADDR=$(ip addr show $DEV | awk '/inet /{ gsub("/.*", ""); print $2}')
+    fi
+    oc cluster up --public-hostname=${IP_ADDR} --routing-suffix=${IP_ADDR}.nip.io --base-dir=$HOME/ocp
+    if [ "$?" -eq 0 ]; then
+      built=true
+      break
+    fi
+    echo "Retrying oc cluster up after failure"
+    oc cluster down
+    sleep 5
+  done
+  echo "OpenShift Cluster Running"
+}
```

Comment:
I think this section can be removed as `oc cluster up` is no longer supported in OCP4